### PR TITLE
Update README with hot deploy

### PR DESCRIPTION
**Pull request by jhadvig**
@mfojtik @bparees PTAL

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -105,5 +105,28 @@ Another option is to make a request directly to `/articles` which will give you
 
 The username/pw used for authentication in this application are openshift/secret.
 
+### Hot Deploy
+
+In order to dynamically pick up changes made in your application source code, you need to make following steps:
+
+- Set the RAILS_ENV variable in the [template](https://github.com/openshift/rails-ex/blob/master/openshift/templates/rails-postgresql.json) to `development` value:
```
Comment (bparees) [inline README.md]: maybe this should be a template parameter?

**Comment by mfojtik** [comment]
@jhadvig can you also document how to switch back to production mode? (oc env?)

**Comment by mfojtik** [comment]
one suggestion, LGTM otherwise.

**Comment by bparees** [comment]
oc env or edit the dc. either way you need to redeploy after.

**Comment by jhadvig** [comment]
@mfojtik suggestion addressed :+1: thanks

File: README.md [hunk 2]
```diff
@@ -105,5 +105,27 @@ Another option is to make a request directly to `/articles` which will give you
...
+
+After you [oc rsh](https://docs.openshift.org/latest/cli_reference/basic_cli_operations.html#troubleshooting-and-debugging-cli-operations) into the running container, your current directory is set to `/opt/app-root/src`, where the source code is located.
+
+To set your application back to the `production` environment you need to:
+
+* remove `RAILS_ENV` environment variable
+
+		$ oc env dc/rails-postgresql-example RAILS_ENV-
+
+* redeploy your application
```
Comment (bparees) [inline README.md]: deploy will happen automatically, if it doesn't it's a bug:
https://github.com/openshift/rails-ex/blob/master/openshift/templates/rails-postgresql.json#L148

Comment (bparees) [inline README.md]: (my comment of "you need to redeploy after" should have said "you need to ensure a redeploy happens after, either manually or via trigger")

i'd update the comment to say "a redeploy will happen automatically due to the configchange trigger" so they understand that it's needed and also why it happened for them.

**Pull request merged**